app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::ItemsController < ApiController
+  skip_before_action :authenticate, only: :create
+
+  def create
+    item = Item.new(item_params)
```

Comment:
Okay, when trying to create an item for the command line using this curl request: 
```
curl -d "item[name]=Dance if you want to" http://localhost:3000/api/users/1/items
```

I get the following error:
```
{"errors":["User can't be blank"]
```
The users create is working properly though. How can I pass in the user in the curl request for the create? Thanks!